utils/styler.py

```diff
@@ -1,12 +1,193 @@
+
+    # Add velocity thresholds style
+    style_manager = lyr.styleManager()
+    lyr.styleManager().removeStyle('Thresholds')
+    if variable == 'u1':
+        default_style_name = style_manager.currentStyle()  # default style name depends on language settings
+        style_name = 'Thresholds'
+        qml_path = STYLES_ROOT / 'tools' / 'animation_toolbar' / 'velocity_thresholds.qml'
+        style = QgsMapLayerStyle()
+        style.readFromLayer(lyr)
+        style_manager.addStyle(style_name, style)
+        style_manager.setCurrentStyle(style_name)
+        (message, success) = lyr.loadNamedStyle(os.path.join(qml_path, qml_path))
+        if not success:  # if style not loaded remove it
+            style_manager.removeStyle(style_name)
+            logger.info("Styling file not succesfully loaded: {fn}".format(fn=qml_path))
+            logger.info(message)
+        style_manager.setCurrentStyle(default_style_name)
+
+    iface.layerTreeView().refreshLayerSymbology(lyr.id())
+    lyr.triggerRepaint()
+
+
+def style_animation_node_current(
+        lyr: QgsVectorLayer,
+        percentiles: List[float],
+        variable: str,
+        cells: bool = False
+):
+    """Applies styling to Animation Toolbar node layer in 'current' mode"""
+
+    # Load basic style settings from qml file
+    if cells:
+        qml_path = STYLES_ROOT / 'tools' / 'animation_toolbar' / 'cell_current.qml'
+    else:
+        qml_path = STYLES_ROOT / 'tools' / 'animation_toolbar' / 'node_current.qml'
+    lyr.loadNamedStyle(str(qml_path))
+    ren = lyr.renderer()
+
+    # Set classes
+    if variable == 's1':
+        class_attribute_str = str('coalesce(result, z_coordinate)')
+    else:
+        class_attribute_str = str('result')
+    lyr.renderer().setClassAttribute(class_attribute_str)
+    ren.deleteAllClasses()
+    nr_classes = len(percentiles) - 1
+    percentiles[0] = -9999  # to make nodes / cells are also visible when dry and z_coordinate < percentile[0]
+    for i in range(nr_classes):
+        ren.addClassLowerUpper(
+            lower=percentiles[i],
+            upper=percentiles[i + 1]
+        )
+    color_ramp = color_ramp_from_data(COLOR_RAMP_OCEAN_HALINE)
+    lyr.renderer().updateColorRamp(color_ramp)
+
+    iface.layerTreeView().refreshLayerSymbology(lyr.id())
+    lyr.triggerRepaint()
+
+
+def style_animation_node_difference(
+        lyr: QgsVectorLayer,
+        percentiles: List[float],
+        variable: str,
+        cells: bool = False
+):
+    """Applies styling to Animation Toolbar node layer in 'difference' mode"""
+
+    # Load basic style settings from qml file
+    if cells:
+        qml_path = STYLES_ROOT / 'tools' / 'animation_toolbar' / 'cell_current.qml'
+    else:
+        qml_path = STYLES_ROOT / 'tools' / 'animation_toolbar' / 'node_difference.qml'
+    lyr.loadNamedStyle(str(qml_path))
+
+    abs_high = max(abs(percentiles[1]), abs(percentiles[-2]))
+    abs_max = max(abs(percentiles[0]), abs(percentiles[-1]))
```

Comment:
What is the difference between "high" and "max"?

Perhaps add a quick comment (or change the variable name)?

Reply:
added a comment